cwe_checker_rs/src/intermediate_representation/expression/builder.rs

```diff
@@ -1,8 +1,47 @@
-use super::{BinOpType, Expression};
+use apint::ApInt;
+
+use super::{BinOpType, CastOpType, Expression, Variable};
 use crate::prelude::*;
 
 /// ## Helper functions for building expressions
 impl Expression {
+    // Shortcut for creating a constant expression from an i64 value
```

Comment:
Doc comments start with `///`, not with `//`. This also applies to the other functions in this module.